sk0-front/src/app/auth/auth.service.ts

```diff
@@ -16,7 +16,7 @@ interface LoginResponse {
 })
 export class AuthService {
   private _isUserLoggedIn = new BehaviorSubject<boolean>(false);
-  private _userName = 'Vendég';
+  private _userName = new BehaviorSubject<string>('');
```

Comment:
Looks good for now, but technically this ons should be rahter null instead of empty string.

Reply:
cool! good idea, the right way!